### DISCUSSION TITLE
Default PTR recoreds

### DISCUSSION THREAD

**Discussion by mm3906078** [coredns/coredns#6211] [Q&A] [answered]
Hi guys
I want to have a default output for all PTR records like this:
```
dig @localhost -x $IP

....
;; ANSWER SECTION:
$IP.in-addr.arpa. 0     IN      PTR     $IP.static.mrsalehi.info.
```

and when I want to set the records overwrite the default answer, I use this config for now:

```
in-addr.arpa.:53 {
    log
    errors
    health
    reload
    file /etc/coredns/records.db
}
```

Does anyone have any idea how I can do this?

**Comment by chrisohaver** [comment, marked as answer]
Take a look at the template plugin.

**Reply by mm3906078** [reply]
Thanks for answering,
I check it and that's great but how I can overwrite it in the file plugin?

my config is like this now:
```
in-addr.arpa.:53 {
    log
    errors
    health
    reload
    file /etc/coredns/records.db
    template IN PTR in-addr.arpa {
        # 1.0.0.127.in-addr.arpa
        match "^(?P<a>[0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])[.](?P<b>[0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])[.](?P<c>[0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])[.](?P<d>[0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])[.]in-addr[.]arpa[.]$"
        answer "{{ .Name }} 86400 IN PTR {{ .Group.d }}-{{ .Group.c }}-{{ .Group.b }}-{{ .Group.a }}.net.example.com"
        fallthrough
    }
}
```

**Comment by chrisohaver** [comment]
I don’t understand the question. 

To elaborate: It looks like that example would achieve your goal. So perhaps I don’t understand your goal correctly.

**Comment by mm3906078** [comment]
I want to place this template in file a 
and a bunch of static records in file b 
record in file b should overwrite the template in file a.
how can I do that?

**Reply by chrisohaver** [reply]
The template plugin is always executed before file plugin. Thus the template plugin will answer all PTR queries, never passing a PTR query to the file plugin.

**Comment by mm3906078** [comment]
I use multiple templates 
thanks
